### PR TITLE
Implementing Ad permalinks mentioned in #17

### PR DESCRIPTION
**Pull request by berislavbabic**
- extracted ad template part into partial.
- Moved title/subtitle into layout
- added basic Ad show page with permalink

### CODE REVIEW

File: app/views/ads/_ad.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<%= link_to ad_path(ad.to_param) do %>
```
Comment (vlado) [inline app/views/ads/_ad.html.erb:1]: Wrapping all this `div`s with `a` tags works ok?

**Review by vlado — COMMENTED** [review]
Looks great!

Left just one small question. Can you post screenshot also ;)

**Comment by berislavbabic** [comment]
@vlado yup, it works

Index page: 
image

Show page:
image

File: app/views/ads/_ad.html.erb (hunk 1 above)
Comment (berislavbabic) [reply app/views/ads/_ad.html.erb:1]: yup, check scheenshots

**Comment by vlado** [comment]
> @vlado yup, it works
> 

On mobile devices also?

**Comment by berislavbabic** [comment]
@vlado I need to add another link because of the maps, gonna refactor this one to just have the 'Podijeli' url as the link in a minute

File: app/views/ads/_ad.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+
+      <%= simple_format ad.description %>
+
+      <hr>
+
+      <div class="columns">
+        <div class="column">Telefon: <b><%= tel_to ad.phone %></b></div>
+        <div class="column">Objavljeno: <b>prije <%= time_ago_in_words ad.created_at %></b></div>
+        <div class="column">Podijeli link na oglas: <%= link_to ad_url(ad.to_param), ad_path(ad.to_param) %></div>
```
Comment (vlado) [inline app/views/ads/_ad.html.erb:41]: I would expect click to copy action here. Maybe we can use that instead of opening the ad?

Comment (vlado) [reply app/views/ads/_ad.html.erb:41]: https://clipboardjs.com

**Comment by berislavbabic** [comment]
Novi screenshot:
image

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -25,16 +25,25 @@ document.addEventListener("turbolinks:load", function(event) {
   let $filterHideLink = $('a#filter-hide');
   let $filterCard = $('#filter-card');
 
+  console.log('bar')
```
Comment (vlado) [inline app/javascript/packs/application.js:28]: Debugging leftover

**Review by vlado — COMMENTED** [review]
Just one debugging leftover and then we are ready to merge.

File: app/javascript/packs/application.js (hunk 3 above)
Comment (berislavbabic) [reply app/javascript/packs/application.js]: oops

**Review by vlado — APPROVED** [review]
🍰

**Pull request merged**